_includes/footer.html

```diff
@@ -1,39 +1,47 @@
 <footer class="c-footer">
   <div class="c-footer__container">
-    <section class="c-footer__award-container">
-      <img src="../assets/Group 2.svg" class="c-footer__award-image" alt="">
-      <p class="c-footer__award-text">named one of the best workplaces to work in ohio for 2016, 2017 and 2018</p>
+    <section class="c-footer__section">
+      <img src="../assets/Group 2.svg" class="c-footer__section-image" alt="">
+      <p class="c-footer__section-text">named one of the best workplaces to work in ohio for 2016, 2017 and 2018</p>
     </section>
     <div class="c-footer__link-lists">
+      <div class="c-footer__link-list-flex-container">
+      <h2 class="c-footer__link-title">pilots</h2>
       <ul class="c-footer__link-list">
-        <h1 class="c-footer__link-title">pilots</h4>
           <li class="c-footer__list-item"><a class="c-footer__link" href="#">First Officer</a></li>
           <li class="c-footer__list-item"><a class="c-footer__link" href="#">Cadet</a></li>
           <li class="c-footer__list-item"><a class="c-footer__link" href="#">Rotor Transition<br>Program</a></li>
       </ul>
+      </div>
+      <div class="c-footer__link-list-flex-container">
```

Comment:
I don't think you need to change this class necessarily 🙂

Reply:
there is a bigger footer_container wrapping everything